src/sass/variables/_variables.scss

```diff
@@ -62,3 +62,5 @@ $font-size-large-1: $font-size-20;
 $font-size-medium: $font-size-16;
 $font-size-regular: $font-size-14;
 $font-size-small: $font-size-12;
+$card-size-default: 132px;
+$card-size-large: 168px;
```

Comment:
Not sure if this is the right place to put these. 
Taking a look at lightbox dialog, we have certain sizes defined in the `:root` of the file
https://github.com/eBay/skin/blob/18.3.0/src/sass/lightbox-dialog/lightbox-dialog.scss#L5-L8
I would follow that pattern

Reply:
I wanted to put it in a place where both the `card` component and the `file-upload` component could use the var. Is there another shared location?

Reply:
Removed these since the sizes will be dynamic and dictated by the parent